code/game/objects/items/extinguisher.dm

```diff
@@ -45,6 +45,19 @@
 	create_reagents(max_water)
 	reagents.add_reagent("water", max_water)
 
+/obj/item/extinguisher/suicide_act(mob/living/carbon/user)
+	if (!safety && (src.reagents.total_volume >= 1))
+		user.visible_message("<span class='suicide'>[user] puts the nozzle to [user.p_their()] mouth. It looks like [user.p_theyre()] trying to extenguish the spark of life!</span>")
+		//playsound(src.loc, 'sound/effects/extinguish.ogg', 75, 1, -3)

```

Comment:
uncomment this and remove .loc

Reply:
whups, that shouldnt even be in there. sound is already handled on line 52

Reply:
> extenguish